src/main/fc/fc_core.c

```diff
@@ -604,158 +601,8 @@ bool processRx(timeUs_t currentTimeUs)
-                && mixerConfig()->mixerMode != MIXER_FLYING_WING
-#endif
-    ) {
-        resetYawAxis();
-    }
-
-    if (throttleCorrectionConfig()->throttle_correction_value && (FLIGHT_MODE(ANGLE_MODE) || FLIGHT_MODE(HORIZON_MODE))) {
-        rcCommand[THROTTLE] += calculateThrottleAngleCorrection(throttleCorrectionConfig()->throttle_correction_value);
-    }
-
-    processRcCommand();
-
-#ifdef USE_NAV
-    if (sensors(SENSOR_GPS)) {
-        if ((FLIGHT_MODE(GPS_HOME_MODE) || FLIGHT_MODE(GPS_HOLD_MODE)) && STATE(GPS_FIX_HOME)) {
-            updateGpsStateForHomeAndHoldMode();
-        }
-    }
-#endif
-
-#ifdef USE_SDCARD
-    afatfs_poll();
-#endif
-
-#ifdef USE_BLACKBOX
-    if (!cliMode && blackboxConfig()->device) {
-        blackboxUpdate(currentTimeUs);
-    }
-#else
-    UNUSED(currentTimeUs);
-#endif
-
-#ifdef USE_TRANSPONDER
-    transponderUpdate(currentTimeUs);
-#endif
-    DEBUG_SET(DEBUG_PIDLOOP, 3, micros() - startTime);
-}
-
-static void subTaskMotorUpdate(timeUs_t currentTimeUs)
-{
-    uint32_t startTime = 0;
-    if (debugMode == DEBUG_CYCLETIME) {
-        startTime = micros();
-        static uint32_t previousMotorUpdateTime;
-        const uint32_t currentDeltaTime = startTime - previousMotorUpdateTime;
-        debug[2] = currentDeltaTime;
-        debug[3] = currentDeltaTime - targetPidLooptime;
-        previousMotorUpdateTime = startTime;
-    } else if (debugMode == DEBUG_PIDLOOP) {
-        startTime = micros();
-    }
-
-    mixTable(currentTimeUs, currentPidProfile->vbatPidCompensation);
-
-#ifdef USE_SERVOS
-    // motor outputs are used as sources for servo mixing, so motors must be calculated using mixTable() before servos.
-    if (isMixerUsingServos()) {
-        writeServos();
-    }
-#endif
-
-    writeMotors();
-
-    DEBUG_SET(DEBUG_PIDLOOP, 2, micros() - startTime);
-}
-
-uint8_t setPidUpdateCountDown(void)
-{
-    if (gyroConfig()->gyro_soft_lpf_hz) {
-        return pidConfig()->pid_process_denom - 1;
-    } else {
-        return 1;
-    }
-}
-
-// Function for loop trigger
-void taskMainPidLoop(timeUs_t currentTimeUs)
-{
-    static uint8_t pidUpdateCountdown = 0;
-
-#if defined(SIMULATOR_BUILD) && defined(SIMULATOR_GYROPID_SYNC)
-    if (lockMainPID() != 0) return;
-#endif
-
-    // DEBUG_PIDLOOP, timings for:
-    // 0 - gyroUpdate()
-    // 1 - pidController()
-    // 2 - subTaskMotorUpdate()
-    // 3 - subTaskMainSubprocesses()
-    gyroUpdate(currentTimeUs);
-    DEBUG_SET(DEBUG_PIDLOOP, 0, micros() - currentTimeUs);
-
-    if (pidUpdateCountdown) {
-        pidUpdateCountdown--;
-    } else {
-        pidUpdateCountdown = setPidUpdateCountDown();
-        subTaskPidController(currentTimeUs);
-        subTaskMotorUpdate(currentTimeUs);
-        subTaskMainSubprocesses(currentTimeUs);
-    }
-
-    if (debugMode == DEBUG_CYCLETIME) {
-        debug[0] = getTaskDeltaTime(TASK_SELF);
-        debug[1] = averageSystemLoadPercent;
-    }
-}
-
+//###########################
```

Comment:
This should probably go.

Reply:
Those were just copy/paste marks so I could keep track of where I moved the code.  I'll delete.